tests/integration/test_updated_flatten_to_tsv_integration.py

```diff
@@ -0,0 +1,94 @@
+"""Integration test for updated_flatten_to_tsv script."""
+
+import subprocess
+import tempfile
+from pathlib import Path
+
+import pytest
+
+script_dir = Path(__file__).parent
+test_dir = script_dir.parent
+root_dir = test_dir.parent
+
+input_dir = test_dir / "input"
+output_dir = test_dir / "output"
+
+SCRIPT_PATH = root_dir / "src" / "dm_bip" / "format_converter" / "updated_flatten_to_tsv.py"
```

Comment:
do you want to update the script name to remove updated based on our Slack chat --> "updated_flatten_to_tsv.py"